imageio/plugins/tifffile.py

```diff
@@ -42,8 +42,20 @@
     By default this setting is inferred from the data shape.
     'contig': last dimension contains samples.
     'planar': third last dimension contains samples.
-resolution_unit : (float, float) or ((int, int), (int, int))
-    X and Y resolution in dots per inch as float or rational numbers.
+resolution_unit : int
+    The resolution unit stored in the TIFF tag. Usually 1 means no/unknown unit,
+    2 means dpi (inch), 3 means dpc (centimeter).
+resolution : ((int, int), (int, int), str)
+    A tuple formatted as (X_resolution, Y_resolution, unit). Both resolutions
+    are given as rational numbers, i.e. as a (numerator, denominator) tuple. Unit
```

Comment:
Would it make more sense to return these as floats, or as [real rational numbers](https://docs.python.org/3/library/fractions.html)?